string robot_kind
string model
string calibration
---
string message
